src/test/java/reposense/parser/CsvParserTest.java

```diff
@@ -85,6 +92,24 @@ public void authorConfig_noSpecialCharacter_success() throws IOException {
         Assert.assertEquals(AUTHOR_CONFIG_NO_SPECIAL_CHARACTER_AUTHORS, config.getAuthorList());
     }
 
+    @Test
+    public void authorConfig_emptyLocation_success() throws ParseException, IOException {
+        RepoConfiguration expectedConfig = new RepoConfiguration("");
+
+        String input = String.format("-config %s", TEST_EMPTY_LOCATION_CONFIG_FOLDER);
+        CliArguments cliArguments = ArgsParser.parse(translateCommandline(input));
```

Comment:
This is a `CsvParser` unit test, no need to bring in the `ArgsParser` component for testing.

Reply:
Referring to the previous method `merge_twoRepoConfigs_success`,  it also uses the `ArgsParser`.  It is used for the constructor of  RepoConfigCsvParser.